src/components/structures/GroupView.js

```diff
@@ -563,12 +566,22 @@ export default React.createClass({
         this._closeSettings();
     },
 
+    _onAction(payload) {
+        switch (payload.action) {
+            // NOTE: close_settings is an app-wide dispatch; as it is dispatched from MatrixChat
```

Comment:
Aren't all dispatches app-wide?

Reply:
Yes. I guess I felt like it was worth mentioning that it's handled in multiple places.

Reply:
ah right, I see what you mean.

Reply:
I'll clarify this.